README.md

```diff
@@ -1,192 +1,144 @@
 
-## Community, discussion, contribution, and support
+The [Cluster API][cluster_api] brings
+declarative, Kubernetes-style APIs to cluster creation, configuration and
+management.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+The API itself is shared across multiple cloud providers allowing for true OpenStack
+hybrid deployments of Kubernetes. It is built atop the lessons learned from
+previous cluster managers such as [kops][kops] and
+[kubicorn][kubicorn].
 
-You can reach the maintainers of this project at:
 
-- [#cluster-api-openstack on Kubernetes Slack](https://kubernetes.slack.com/messages/cluster-api-openstack)
-- [SIG-Cluster-Lifecycle Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+## Launching a Kubernetes cluster on OpenStack
 
-### Code of conduct
+Check out the [getting started guide](./docs/getting-started.md) for launching a cluster on OpenStack. 
 
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+## Features
+
+- Native Kubernetes manifests and API
+- Choice of Linux distribution (as long as a current cloud-init is available)
+- Support for single and multi-node control plane clusters
+- Deploy clusters with and without LBaaS available
+- Support for security groups
+- Doesn't use SSH for bootstrapping nodes
 
 ------
 
-## Compatibility with Cluster API, Kubernetes and OpenStack Versions
+## Compatibility with Cluster API and Kubernetes Versions
 
 This provider's versions are compatible with the following versions of Cluster API:
 
-||Cluster API v1alpha1 (v0.1)|
-|-|-|
-|OpenStack Provider v1alpha1 (ea309e7f)|✓|
+||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (v0.2)|
+|-|-|-|
+|OpenStack Provider v1alpha1 (release-0.1 branch)|✓||
+|OpenStack Provider v1alpha2 (v0.2)||✓|
 
 This provider's versions are able to install and manage the following versions of Kubernetes:
 
-||Kubernetes 1.13.5+|Kubernetes 1.14|Kubernetes 1.15|
+||Kubernetes 1.13|Kubernetes 1.14|Kubernetes 1.15|
 |-|-|-|-|
-|OpenStack Provider v1alpha1 (ea309e7f)|✓|✓|✓|
-
-Kubernetes control plane and Kubelet versions are defined in `spec.versions.controlPlane` and `spec.versions.kubelet` of `cmd/clusterctl/examples/openstack/machines.yaml.template` respectively.
-You can generate `cmd/clusterctl/examples/openstack/out/machines.yaml` by running the `generate-yaml.sh` from the template and change the versions if you want.
-
-**NOTE**: Because the user is able to customize any `user-data`, it is also possible to deploy older versions.
-But we won't provide any examples or working templates. See [user-data in the examples](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/tree/master/cmd/clusterctl/examples/openstack/provider-component/user-data).
+|OpenStack Provider v1alpha1 (release-0.1 branch)|✓|✓|✓|
+|OpenStack Provider v1alpha2 (v0.2)|||✓|
```

Comment:
master branch or v0.2 branch will be better? v0.2 seems a tag..

Reply:
Same as above 0.2 stands for all our 0.2.x releases (which will be tags)

Reply:
ditto..